### PR TITLE
change the getHotelReviews to getDataReviews

### PR DESCRIPTION
**Pull request by Achraf-El-khaier**
## Requirement
change the getHotelReviews to getDataReviews

## Screenshots <sub>[Optional]</sub>

*Visualizations of the functionality in a test environment. (Can be Postman)*
- `Answer here`

### Testing Plan

- *Is it uploaded to a scope? Which one?.*
  - [ ] Dev
  - [ ] Stage
  - [x] Other
    - `Answer here`

- *Which testing user should be used? (User, password, etc.).*
  - `Answer here`

- *Indicate the step sequence to reproduce this functionality, and every context information that help us to test it.*
  - 1 - `Description of step one`
    - ```bash
        curl -X GET to.reach.step.one
      ```
  - 2 - `Description of step two`
    - ```JSON
        {"json":"With expected results"} 
      ```
- [ ] QA should be informed about this PR?

## Documentation <sub>[Optional]</sub>
*If exists, add a confluence link.*
- `Answer here`

- [ ] The documentation in the functions of this PR is updated

## Rollout <sub>[Optional]</sub>

*Date when this functionality should be uploaded to production*
- [ ] NEXT_DEPLOY
- [ ] Release Nº `X`

### Communication

- [x] This PR was shared in the Slack chat?
- [ ] The customers should be informed about these changes.

### Wordings

- [ ] Does it have wording changes?

### Technical Track 
- [ ] Maintains or increases the test coverage
- [ ] Adds a refactor
- [ ] Adds metrics
- [ ] Adds logs

### CODE REVIEW

**Review by santiagomoranlabat — APPROVED** [review]

File: test/test.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ describe('LiteAPI SDK Test Suite', function() {
   });
 
   it('should retrieve hotel reviews by ID', async function() {
-    const result = await liteApi.getHotelReviews('lp1897', 5, true);
+    const result = await liteApi.getDataReviews('lp1897', 5, true);
```
Comment (squallstar) [inline test/test.js:146]: Why this change?
